cmd/example-mysql/main.go

```diff
@@ -120,19 +125,18 @@ func main() {
 	}
 }
 
-// parseTileLevelIndex takes level and index in string, validates and returns them in uint64.
-// Note that this doesn't return any of the partial tile component.
+// parseTileLevelWidthIndex takes level and index in string, validates and returns the level, index and width in uint64.
 //
 // Examples:
 // "/tile/0/x001/x234/067" means level 0 and index 1234067 of a full tile.
 // "/tile/0/x001/x234/067.p/8" means level 0, index 1234067 and width 8 of a partial tile.
-func parseTileLevelIndex(level, index string) (uint64, uint64, error) {
+func parseTileLevelIndexWidth(level, index string) (uint64, uint64, uint64, error) {
```

Comment:
Can we have some unit tests for this? I'll approve this and leave it up to you whether you want to put them in this PR, or a follow-up.

Reply:
Sure, let me do that in a follow-up pull request today.